package.json

```diff
@@ -69,5 +69,10 @@
     "engines": {
         "node": "24",
         "pnpm": "10.17.0"
+    },
+    "pnpm": {
+        "overrides": {
+            "@apollo/server-plugin-landing-page-graphql-playground>@apollo/server": "^5.1.0"
+        }
```

Comment:
The following error now occurs during the install:

```
demo/api
└─┬ @nestjs/apollo 13.2.1
  └─┬ @apollo/server-plugin-landing-page-graphql-playground 4.0.1
    └── ✕ unmet peer @apollo/server@^4.0.0: found 5.1.0
```

This is a problem caused by `@nestjs/graphql` because there is no v5 version for `@apollo/server-plugin-landing-page-graphql-playground`. Still it's in the dependencies: https://github.com/nestjs/graphql/blob/v13.2.1/packages/apollo/package.json#L41

Apparently, the plan is to remove it in v14 but not earlier: https://github.com/nestjs/graphql/issues/3458#issuecomment-2801348664

I fixed it by overriding the peer dependency